internal/infrastructure/db/migrations/20231113225121_create_template_dd.sql

```diff
@@ -0,0 +1,29 @@
+-- +goose Up
+-- +goose StatementBegin
+SELECT 'up SQL query';
+
+CREATE TABLE IF NOT EXISTS template_device_definitions (
+    id BIGSERIAL,
+    device_definition_id BIGINT NOT NULL,
+    template_name TEXT REFERENCES templates(template_name) NOT NULL,
+    year_start INTEGER NOT NULL,
+    year_end INTEGER NOT NULL,
+    make_slug TEXT NOT NULL,
+    model_whitelist TEXT[] NOT NULL, -- array of model slugs, corresponding to ModelWhitelist in TemplateVehicle
+    hierarchy_level INTEGER NOT NULL, -- Determines the specificity level in the hierarchy
```

Comment:
no need for these - as this is a specific mapping from a template to a device definition eg. https://admin.team.dimo.zone/device-definitions/2M6v4uRkXvhiBqWe8YCSd0G6G7s
which already has the MMY